### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_EditAttachments_HighContrast: In High Contrast Mode, 'Close' Button is not clearly visible.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#4011]
**Environment Details:**
Application Name: AzureNotebooks_Jupyter
Login URL: notebooks.azure.com
Browser Details:
Edge: Microsoft Edge 42.17134.1.0
Chrome: Version 68.0.3440.106 
Operating System:Windows10

**Steps to reproduce:**
1.Login with the appropriate credentials to azure notebook account.    
2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library
3.Click on Run or right click and select Run to run the notebook.
4.Navigate to any file present under Files tab-->Hit enter-->View Menu-->Cell Toolbar-->Attachments. Click on Edit Attachments button appeared on page.
5.Verify the Popup under HighContrast(black) mode.

**Actual**
'Close' button is not properly visible under high contrast mode.

**Note:
Same issue observed on below pages:**
1)RenameConfirmationPopUp: Navigate to run button & hit enter --> select any file --> navigate to Rename button & hit enter.
2)MoveConfirmationPopup: Navigate to run button & hit enter --> select any file --> navigate to Move button & hit enter.
3)EditMetada (Navigate to any file present under Files tab-->Hit enter-->View Menu-->Cell Toolbar-->Edit Metadata. Click on Edit Metadata button appeared on page.)

**Expected**
'Close' button should be properly visible even under high contrast mode.

**User Impact**
Low vision user will not be able to identify the close button easily.
**Attachment:**
![1329741_a11y_azurenotebooks_jupyter_editattachments_highcontrast](https://user-images.githubusercontent.com/39204909/45818557-b0946e00-bcff-11e8-8a53-398a8ada1d8c.png)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
